ad/movement/trusts.md

```diff
@@ -28,6 +28,16 @@ Simply establishing a trust relationship does not automatically grant access to
 A trust relationship allows users in one domain to **authenticate** to the other domain's resources, but it does not automatically grant access to them. Access to resources is controlled by permissions, which must be granted explicitly to the user in order for them to access the resources.
 {% endhint %}
 
+### Global Catalog
+The global catalog is a partial copy of all objects in an Active Directory forest, meaning that some object properties (but not all) are contained within it. This data is replicated among all domain controllers marked as global catalogs for the forest. One point of the global catalog is to allow for object searching and deconfliction quickly without the need for referrals to other domains (more information here).
+
+The initial global catalog is generated on the first domain controller created in the first domain in the forest. The first domain controller for each new child domain is also set as a global catalog by default, but others can be added.
+
+The GC allows both users and applications to find information about any objects in ANY domain in the forest. GC is a feature that is enabled on a domain controller and performs the following functions:
```

Comment:
`GC is a feature that is enabled on a domain controller and performs [...]` -> `The Global Catalog performs [...]` (you already said GC can be marked and enabled on DCs

Reply:
Completed!